### PR TITLE
Add --script-readable to env and env docker

### PR DESCRIPTION
**Pull request by bexelbie**

### CODE REVIEW

**Comment by bexelbie** [comment]
Works toward #70

**Comment by bexelbie** [comment]
```
> bundle exec vagrant service-manager env --script-readable
DOCKER_HOST=tcp://10.90.2.2:2376
DOCKER_CERT_PATH='/home/bexelbie/Repositories/projectatomic/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker'
DOCKER_TLS_VERIFY=1
DOCKER_API_VERSION=1.20
OPENSHIFT_URL=https://10.90.2.2:8443
OPENSHIFT_WEB_CONSOLE=https://10.90.2.2:8443/console

> bundle exec vagrant service-manager env docker --script-readable
DOCKER_HOST=tcp://10.90.2.2:2376
DOCKER_CERT_PATH='/home/bexelbie/Repositories/projectatomic/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker'
DOCKER_TLS_VERIFY=1
DOCKER_API_VERSION=1.20

> bundle exec vagrant service-manager env openshift --script-readable
OPENSHIFT_URL=https://10.90.2.2:8443
OPENSHIFT_WEB_CONSOLE=https://10.90.2.2:8443/console
```

**Comment by brgnepal** [comment]
@bexelbie , I don't think this piece is script readable:

```
Configured services:
docker - running
openshift - running
kubernetes - stopped
```

**Comment by bexelbie** [comment]
@budhrg I believe that is fixed already

File: lib/vagrant-service-manager/command.rb [hunk 1]
```diff
@@ -108,8 +112,10 @@ def check_if_a_service_is_running?(service)
         end
       end
 
-      def print_all_provider_info
-        @env.ui.info I18n.t('servicemanager.commands.env.nil')
+      def print_all_provider_info(script_readable = false)
+        if !script_readable
```
Comment (brgnepal) [inline lib/vagrant-service-manager/command.rb]: Please prefer `unless` instead of `if  !` condition

**Comment by navidshaikh** [comment]
Here are the commands and their output using this PR

```
➜  vagrant service-manager env
Configured services:
docker - running
openshift - running
kubernetes - stopped

docker env:
# Set the following environment variables to enable access to the
# docker daemon running inside of the vagrant virtual machine:
export DOCKER_HOST=tcp://172.28.128.8:2376
export DOCKER_CERT_PATH=/home/nshaikh/work/src/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker
export DOCKER_TLS_VERIFY=1
export DOCKER_API_VERSION=1.21
# run following command to configure your shell:
# eval "$(vagrant service-manager env docker)"

openshift env:
You can access the OpenShift console on: https://172.28.128.8:8443/console
To use OpenShift CLI, run: oc login https://172.28.128.8:8443
```

```
➜  vagrant service-manager env --script-readable
DOCKER_HOST=tcp://172.28.128.8:2376
DOCKER_CERT_PATH='/home/nshaikh/work/src/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker'
DOCKER_TLS_VERIFY=1
DOCKER_API_VERSION=1.21
OPENSHIFT_URL=https://172.28.128.8:8443
OPENSHIFT_WEB_CONSOLE=https://172.28.128.8:8443/console
```

```
➜  vagrant service-manager env docker --script-readable
DOCKER_HOST=tcp://172.28.128.8:2376
DOCKER_CERT_PATH='/home/nshaikh/work/src/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker'
DOCKER_TLS_VERIFY=1
DOCKER_API_VERSION=1.21
```

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**